bodhi/server/notifications.py

```diff
@@ -64,16 +64,19 @@ def send_fedmsgs_after_commit(session):
         session (sqlalchemy.orm.session.Session): The session that was committed.
     """
     if 'fedmsg' in session.info:
+        _log.info('Emitting all queued fedmsgs for %r', session)
```

Comment:
I wonder if we should make this debug? This could make the logs pretty noisy since so many actions in Bodhi generate fedmsgs.